calcrule_social_protection/utils.py

```diff
@@ -0,0 +1,7 @@
+import random
+
+
+def generate_unique_code(length):
```

Comment:
That is temporary solution until generic code generation is done. @dborowiecki @jdolkowski @malinowskikam .

Reply:
Please add a check that confirms that the code is not duplicated in the database. For the time being we should be fine with this approach but this will give us additional security layer.